VideoApp/VideoApp/Stores/AppSettings/AppSettingsStore.swift

```diff
@@ -35,12 +35,13 @@ protocol AppSettingsStoreWriting: LaunchStore {
     var lowRenderDimensions: VideoDimensionsName { get set }
     var standardRenderDimensions: VideoDimensionsName { get set }
     var highRenderDimensions: VideoDimensionsName { get set }
+    var remoteRoomType: CommunityCreateTwilioAccessTokenResponse.RoomType? { get set }
```

Comment:
I'm bending the rules a little bit by using this `RoomType` from the API layer directly and not creating a new type for settings. But all we do with this is store it so we can track when there is a change on the server (in which case we automatically update `videoCodec`). The only thing a new type would buy me is I could then make `CommunityCreateTwilioAccessTokenResponse.RoomType` only conform to `Decodable` instead of `Codable` (`Decodable` + `Encodable`). But that didn't really seem worth it given how this setting is used and it's not like we're going to separate API and settings code into separate modules.

If we do more remote config in the future I think it would make sense at some point to create a `RemoteConfigStore` completely separate from settings but that seemed overkill right now.

Reply:
Works for me.